### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by DiggyDre** [bmaltais/kohya_ss#628]
This won't install for me. I'm on a Windows 10 with the RTX-4070ti. The setup won't even run.

**Comment by bmaltais** [comment]
Not sure why this is the case for some users. When you type .\setup.bat it does not run the installation? Hard to troubleshoot without the error message that follows running .\setup.bat

**Comment by CreateLab** [comment]
cant install tensorflow 2.10 or earlier 
```
ERROR: Could not find a version that satisfies the requirement tensorflow==2.10.1 (from versions: 2.12.0rc0, 2.12.0rc1, 2.12.0)
ERROR: No matching distribution found for tensorflow==2.10.1

[notice] A new release of pip available: 22.3 -> 23.1
[notice] To update, run: python.exe -m pip install --upgrade pip
ERROR: Could not find a version that satisfies the requirement xformers==0.0.17 (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.0.12, 0.0.13, 0.0.16rc424, 0.0.16rc425, 0.0.16)
ERROR: No matching distribution found for xformers==0.0.17
```
windows platform
